pyproject.toml

```diff
@@ -0,0 +1,16 @@
+[tool.ruff]
+target-version = "py310"
+select = ["E", "F", "I", "UP"]
```

Comment:
This configuration looks for lint errors (E/F), isort errors (I), and pyupgrade errors (UP).